mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -23,12 +24,20 @@ def cross_entropy(pred,
         avg_factor (int, optional): Average factor that is used to average
             the loss. Defaults to None.
         class_weight (list[float], optional): The weight for each class.
+        ignore_index (int | None): The label index to be ignored.
+            If None, it will be set to default value. Default: -100.
```

Comment:
I am confused about this default value

Reply:
-100 is the default value of ignore_index in F.cross_entropy_loss.